shell/platform/embedder/embedder_thread_host.cc

```diff
@@ -120,6 +125,22 @@ EmbedderThreadHost::CreateEmbedderManagedThreadHost(
     return nullptr;
   }
 
+#if OS_LINUX
+  thread_host.gpu_thread->GetTaskRunner()->PostTask([]() {
```

Comment:
Can the application achieve the same effect via the `FlutterEngineRunTask` and `FlutterEnginePostRenderThreadTask` embedder APIs?